packages/app/components/settings/add-email.tsx

```diff
@@ -27,25 +28,26 @@ export const AddEmailModal = () => {
 
   const submitEmail = useCallback(
     async (email: string) => {
+      rudder?.track("Button Clicked", {
+        name: "Login with email",
+      });
+
       try {
         const did = await magic.auth.loginWithMagicLink({ email });
         if (did) {
           await addEmail(email, did);
           router.pop();
         }
       } catch (error) {
-        console.log("Error:", error);
+        console.error(error);
```

Comment:
nit: we can use Logger.error(). Right now it doesn't do anything but in future we can avoid logging on prod and send it to sentry.

Reply:
yeah agreed! use  `Logger.error` will be better!

Reply:
OK good idea, let's switch all `console.error` to `Logger.error` in a separate PR